jib-cli/src/main/java/com/google/cloud/tools/jib/cli/war/WarFiles.java

```diff
@@ -77,12 +77,24 @@ private static List<String> computeEntrypoint(
       CommonContainerConfigCliOptions commonContainerConfigCliOptions)
       throws InvalidImageReferenceException {
     List<String> entrypoint = commonContainerConfigCliOptions.getEntrypoint();
+    String baseImage = fetchBaseImage(commonContainerConfigCliOptions);
     if (!entrypoint.isEmpty()) {
       return entrypoint;
     }
     if (commonContainerConfigCliOptions.isJettyBaseimage()) {
+      // If using jetty 12 or later, then specify deploy module. See
```

Comment:
Where are we detecting that it's version 12 of Jetty?

Reply:
That's a good question. The current default in Jib is `jetty:latest` and this logic makes the assumption that the [latest docker image](https://hub.docker.com/layers/library/jetty/latest/images/sha256-eceaca283f0cd3a51741801cccecc38b279cf8153ad940b332f0d3ec2b7f9c8c?context=explore) will be v12 or greater now (as seen by the `ENV JETTY_VERSION=12.0.7` in the image's layer) now that jetty 10 and jetty 11 have reached [EOL](https://github.com/jetty/jetty.project/issues/10485). 

The key logic for determining the `tag` lies in [ContainerBuilders#create](https://github.com/GoogleContainerTools/jib/blob/6df635ee51bc466c961cdeea23d84e84e87ba4b0/jib-cli/src/main/java/com/google/cloud/tools/jib/cli/ContainerBuilders.java#L69) which is called by WarFiles#toJibContainerBuilder. This class then calls `ImageReference.parse(baseImage)` which parses the registry, repository and tag from baseImage. If a tag isn't specified (which is the case here) then `latest` is set as the default.

Reply:
In that case, it might be worthlwhile to clarify the comment. For instance, `// Since we're using Jetty 12 or later...`.

Reply:
Good idea, done.